server/channels/api4/preference_test.go

```diff
@@ -257,6 +257,40 @@ func TestUpdatePreferences(t *testing.T) {
 	CheckUnauthorizedStatus(t, resp)
 }
 
+func TestUpdatePreferencesOverload(t *testing.T) {
+	th := Setup(t).InitBasic()
+	defer th.TearDown()
+	client := th.Client
+
+	th.LoginBasic()
+	user1 := th.BasicUser
+
+	category := model.NewId()
+	preferences1 := model.Preferences{}
```

Comment:
I'd prefer each test to have its own set of preferences to avoid carrying over the state from one test to another.